src/qibo/quantum_info/metrics.py

```diff
@@ -8,7 +8,7 @@ def purity(state):
     """Purity of a quantum state :math:`\\rho`, which is given by :math:`\\text{Tr}(\\rho^{2})`.
 
     Args:
-        state: statevector or density matrix.
+        state (ndarray): statevector or density matrix.
```

Comment:
I always write `np.ndarray` for type in the documentation, but maybe that's wrong? Just checking here.

Reply:
It could be a `cupy.ndarray` (or even a `tensorflow.Tensor`, now that I think about it)